pyproject.toml

```diff
@@ -43,9 +43,9 @@ django-auth-ldap = {version = "~4.1.0", optional = true}
 # Caching with Redis
 django-cacheops = "~6.0"
 # Base functionality for task scheduling with Celery
-django-celery-beat = "~2.2.1"
+django-celery-beat = "~2.3.0"
 # Management of app configuration via the Django admin UI
-django-constance = {version = "~2.8.0", extras = ["database"]}
+django-constance = {version = "~2.9.0", extras = ["database"]}
```

Comment:
We may not need a custom `ConstanceDatabaseAppConfig` class (`nautobot.core.apps`) any more as it looks like Constance has fixed the issue that necessitated it.